web/pages/link/[slug].tsx

```diff
@@ -33,14 +33,20 @@ export default function ClaimPage() {
       <div className="mx-auto max-w-xl">
         <Title text={`Claim M$${manalink.amount} mana`} />
         <ManalinkCard
-          defaultMessage={fromUser?.name || 'Enjoy this mana!'}
+          user={user}
+          defaultMessage={`from ${fromUser?.name}`}
```

Comment:
As mentioned IRL: might be better to actually write in the `from Inga Wei` into the database, instead of having special behavior when the message isn't set.